google/cloud/security/scanner/audit/org_rules_engine.py

```diff
@@ -269,22 +340,44 @@ def add_rule(self, rule_def, rule_index):
                                 bindings=rule_def.get('bindings', []),
                                 mode=rule_def.get('mode'))
 
+                    applies_to = resource.get('applies_to')
+
                     # See if we have a mapping of the resource and rule
-                    resource_rules = self.resource_rules_map.get(gcp_resource)
+                    resource_rules = self.resource_rules_map.get(
+                        (gcp_resource, applies_to))
 
                     # If no mapping exists, create it.
                     if not resource_rules:
                         resource_rules = ResourceRules(
                             resource=gcp_resource,
-                            applies_to=resource.get('applies_to'),
+                            applies_to=applies_to,
                             inherit_from_parents=rule_def.get(
                                 'inherit_from_parents', False))
-                        self.resource_rules_map[gcp_resource] = resource_rules
+                        self.resource_rules_map[
+                            (gcp_resource, applies_to)] = resource_rules
```

Comment:
Above, around line 346, you could set rules_map_key = (gcp_resource, applies_to), then for the get on line 346 and here you can use that variable, which is shorter. Another option is use () to move resource_rules to the next line.

self.resource_rules_map[<blah>] = (
    resource_rules)